src/App.jsx

```diff
@@ -8,30 +8,37 @@ import Home from "./components/Home";
 import Login from "./components/Login";
 import Navigate from "./components/Navigate";
 import Register from "./components/Register";
-import SiteNav from "./components/SiteNav";
 import NotFound from "./components/NotFound";
+import NavBar from "./components/NavBar";
 
 function App() {
-  return (
-    <Fragment>
-      <SiteNav />
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+
+  const handleLogin = () => {
+    setIsLoggedIn(true);
+  };
+
+  const handleLogout = () => {
+    setIsLoggedIn(false);
+  };
 
-      <BrowserRouter>
+  return (
+    <BrowserRouter>
+      <Fragment>
+        <NavBar isLoggedIn={isLoggedIn} onLogout={handleLogout} />
         <Routes>
           <Route path="/" element={<Home />} />
-          <Route path="/aboutUs" element={<AboutUs />} />
+          <Route path="/about-us" element={<AboutUs />} />
           <Route path="/calendar" element={<Calendar />} />
-          <Route path="/login" element={<Login />} />
+          <Route path="/login" element={<Login onLogin={handleLogin} />} />
```

Comment:
This code should be handled by the login component that was made. We don't want the navbar to log the user in.